shared/js/url_test/url_test.js

```diff
@@ -2,14 +2,7 @@
 // Useful as a workaround for CORS security to test access to an origin.
 function testImageAccess(url, successCallback, failureCallback, timeoutMs) {
   timeoutMs = typeof timeoutMs !== 'undefined' ?  timeoutMs : 5000;
-  var img = document.createElement('img');
-  img.width = 1;
-  img.height = 1;
-  img.style.position = 'absolute';
-  img.style.bottom = 0;
-  img.style.width = 1;
-  img.style.height = 1;
-  img.style.visibility = 'hidden';
+  var img = new Image();
```

Comment:
What does this change do?

Haven't tested this non-dom-included load error detection technique myself across all browsers. Assuming it works, seems fine though.